test-integration/scripts/11-generate-config.sh

```diff
@@ -0,0 +1,187 @@
+    moveEntity Artist
+
+elif [[ "$JHI_ENTITY" == "cassandra" ]]; then
+    moveEntity CassBankAccount
+
+    moveEntity CassTestEntity
+    moveEntity CassTestMapstructEntity
+    moveEntity CassTestServiceClassEntity
+    moveEntity CassTestServiceImplEntity
+
+elif [[ "$JHI_ENTITY" == "micro" ]]; then
+    moveEntity MicroserviceBankAccount
+    moveEntity MicroserviceOperation
+    moveEntity MicroserviceLabel
+
+    moveEntity FieldTestEntity
+    moveEntity FieldTestMapstructAndServiceClassEntity
+    moveEntity FieldTestServiceClassAndJpaFilteringEntity
+    moveEntity FieldTestServiceImplEntity
+    moveEntity FieldTestInfiniteScrollEntity
+    moveEntity FieldTestPaginationEntity
+
+elif [[ "$JHI_ENTITY" == "uaa" ]]; then
+    moveEntity FieldTestEntity
+    moveEntity FieldTestMapstructAndServiceClassEntity
+    moveEntity FieldTestServiceClassAndJpaFilteringEntity
+    moveEntity FieldTestServiceImplEntity
+    moveEntity FieldTestInfiniteScrollEntity
+    moveEntity FieldTestPaginationEntity
+
+elif [[ "$JHI_ENTITY" == "sqllight" ]]; then
+    moveEntity BankAccount
+    moveEntity Label
+    moveEntity Operation
+
+elif [[ "$JHI_ENTITY" == "sqlfull" ]]; then
+    moveEntity BankAccount
+    moveEntity Label
+    moveEntity Operation
+    moveEntity Place
+    moveEntity Division
+
+    moveEntity FieldTestEntity
+    moveEntity FieldTestMapstructAndServiceClassEntity
+    moveEntity FieldTestServiceClassAndJpaFilteringEntity
+    moveEntity FieldTestServiceImplEntity
+    moveEntity FieldTestInfiniteScrollEntity
+    moveEntity FieldTestPaginationEntity
+    moveEntity FieldTestEnumWithValue
+
+    moveEntity TestEntity
+    moveEntity TestMapstruct
+    moveEntity TestServiceClass
+    moveEntity TestServiceImpl
+    moveEntity TestInfiniteScroll
+    moveEntity TestPagination
+    moveEntity TestManyToOne
+    moveEntity TestManyToMany
+    moveEntity TestManyRelPaginDTO
+    moveEntity TestOneToOne
+    moveEntity TestCustomTableName
+    moveEntity TestTwoRelationshipsSameEntity
+    moveEntity SuperMegaLargeTestEntity
+
+    moveEntity EntityWithDTO
+    moveEntity EntityWithPaginationAndDTO
+    moveEntity EntityWithServiceClassAndPagination
+    moveEntity EntityWithServiceClassPaginationAndDTO
+    moveEntity EntityWithServiceImplAndDTO
+    moveEntity EntityWithServiceImplAndPagination
+    moveEntity EntityWithServiceImplPaginationAndDTO
+
+    moveEntity MapsIdParentEntityWithoutDTO
+    moveEntity MapsIdChildEntityWithoutDTO
+    moveEntity MapsIdParentEntityWithDTO
+    moveEntity MapsIdChildEntityWithDTO
+    moveEntity MapsIdUserProfileWithDTO
+
+    moveEntity JpaFilteringRelationship
+    moveEntity JpaFilteringOtherSide
+
+    #-------------------------------------------------------------------------------
+    # Generate jdl entities
+    #-------------------------------------------------------------------------------
+    jhipster --no-insight jdl "$JHI_SAMPLES"/jdl-entities/*.jdl --json-only
```

Comment:
Using JDL for generating entities is nice, but I'd prefer another approch:
- I'd remove this part for sqlfull
- use another variable: JHI_JDL_NAME
- use another if block:

```
if [[ "$JHI_ENTITY" == "jdl" && "$JHI_JDL_NAME" != "" ]]; then
    jhipster --no-insight jdl "$JHI_SAMPLES"/jdl-entities/$JHI_JDL_NAME.jdl --json-only
elif
```

So it will improve our current CI, don't have any impact for daily builds, blueprints, etc.

Reply:
`"$JHI_ENTITY" == "jdl"` is used for generating the entire project using jdl, it would make this PR useless.